doc/api/errors.md

```diff
@@ -2534,6 +2528,12 @@ removed: v10.0.0
 
 Used when a TLS renegotiation request has failed in a non-specific way.
 
+<a id="ERR_TLS_REQUIRED_SERVER_NAME"></a>
+### `ERR_TLS_REQUIRED_SERVER_NAME`
```

Comment:
I believe we have an existing error code for dealing with a missing argument

Reply:
@jasnell This is the section for legacy error codes that are no longer in use